### ISSUE TITLE
Add support for Q1 HE

### ISSUE THREAD

**Issue by yumlevi** [Keychron/qmk_firmware#268]
### Issue Description

Hello,
Please add support for this product.

Thanks.

**Comment by PeterMortensen** [comment]
What do you mean by "support"? Publish the source code? Or something else?

**Comment by yumlevi** [comment]
Yes, publish the source code, right now we can only edit through the extremely lacking web app.

**Comment by lokher** [comment]
https://github.com/Keychron/qmk_firmware/tree/hall_effect_playground/keyboards/keychron/q1_he

**Comment by jasondamour** [comment]
@lokher any plans to merge this branch into master or upstream? Keychron's fork is falling behind QMK
